packages/devtools_app/lib/src/config_specific/drag_and_drop/drag_and_drop.dart

```diff
@@ -18,17 +18,15 @@ abstract class DragAndDropManager {
     init();
   }
 
-  static DragAndDropManager get instance => _instance ?? DragAndDropManager();
+  static DragAndDropManager get instance => _instance;
 
-  static set instance(value) => _instance = value;
-
-  static DragAndDropManager? _instance;
+  static late final DragAndDropManager _instance = DragAndDropManager();
 
   final _dragAndDropStates = <DragAndDropState>{};
 
   DragAndDropState? activeState;
 
-  void init();
+  void init() {}
```

Comment:
make this abstract instead of empty. You can make the desktop override empty.

Reply:
What's the point to obligate descendants to implement it?

Reply:
done